### PR TITLE
Update OpenAPI generator and the client

### PR DESCRIPTION
**Pull request by palnabarun**
This PR also:
1. Remove tests since they are stubs
2. Updates the CHANGELOG with changes from 1.16.9 to 1.16.11
3. Updates the CHANGELOG with changes from python-base
4. Update the hotfix script

Follow ups:
- Update the release documentation

### CODE REVIEW

**Comment by palnabarun** [comment]
There is a test failure in the base submodule:
```
        self.assertEqual(expected_token,
>                        config.auth_settings()['BearerToken']['value'])
E       KeyError: 'BearerToken'

kubernetes/base/config/kube_config_test.py:1373: KeyError
```

File: kubernetes/test/test_admissionregistration_api.py [hunk 1]
```diff
@@ -1,39 +0,0 @@
-# coding: utf-8
```
Comment (roycaihw) [inline kubernetes/test/test_admissionregistration_api.py:1]: Upstream issue: https://github.com/OpenAPITools/openapi-generator/issues/5377

**Comment by roycaihw** [comment]
> There is a test failure in the base submodule

I checked python-base and found there was one pycodestyle failure merged accidentally. I sent a fix: https://github.com/kubernetes-client/python-base/pull/200. 

The error in https://github.com/kubernetes-client/python/pull/1188#issuecomment-647630738 looks different though. I created an experimental PR https://github.com/kubernetes-client/python/pull/1191 that only bumps python-base, to help us better identify if the bug is on our side or introduced by openapi-generator.

**Comment by roycaihw** [comment]
The tests passed in #1191. Looks like the `KeyError` is related to openapi-generator change.

**Comment by roycaihw** [comment]
The test failure 
```
    def test_auth_settings_calls_get_api_key_with_prefix(self):
        expected_token = 'expected_token'
    
        def fake_get_api_key_with_prefix(identifier):
            self.assertEqual('authorization', identifier)
            return expected_token
        config = Configuration()
        config.get_api_key_with_prefix = fake_get_api_key_with_prefix
        self.assertEqual(expected_token,
>                        config.auth_settings()['BearerToken']['value'])
E       KeyError: 'BearerToken'
```
is related to the upstream change https://github.com/OpenAPITools/openapi-generator/pull/4988. In the test we faked a Configuration with a customized `get_api_key_with_prefix`, but we didn't set the `api_key` thus the returned attribute was empty. We should append `authorization` to `api_key` of the fake Configuration in the test.

**Comment by palnabarun** [comment]
/reopen

(the bot mistakenly closed this due to https://github.com/kubernetes-client/python-base/pull/202 merging)

**Comment by palnabarun** [comment]
The build is again failing due to a regression caused in https://github.com/kubernetes-client/python-base/pull/133.

Ref: https://travis-ci.org/github/kubernetes-client/python-base/builds/708448505

**Comment by palnabarun** [comment]
https://github.com/kubernetes-client/python-base/pull/204 will fix the test failure.

**Comment by palnabarun** [comment]
After https://github.com/kubernetes-client/python-base/pull/204 got merged, the tests are passing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -41,9 +41,34 @@ Kubernetes API Version: 1.16.11
 
 **New Feature:**
 
+- Adds the ability to load kubeconfig from a dictionary [kubernetes-client/python-base#195](https://github.com/kubernetes-client/python-base/pull/195)
 - Allow incluster to accept pass-in config [kubernetes-client/python-base#193](https://github.com/kubernetes-client/python-base/pull/193)
 - Set expiration on token of incluster config and reload the token if it expires [kubernetes-client/python-base#191](https://github.com/kubernetes-client/python-base/pull/191)
 
+**Bug Fix:**
+
+- Fixes a bug in loading kubeconfig when there are no users in ttthe config [kubernetes-client/python-base#198](https://github.com/kubernetes-client/python-base/pull/198)
```
Comment (yliaog) [inline CHANGELOG.md:50]: typo ttthe

Comment (palnabarun) [reply CHANGELOG.md]: Thanks for the catch. 👍 It is resolved now.

**Pull request merged**